translations/README-en.md

```diff
@@ -0,0 +1,218 @@
+![Logotype](../logo.png)
+
+> Team Lead is a ❄️, because in each company they are unique and distinct.
+
+- [Roadmap](#roadmap)
+- [Telegram chat TL Bootcamp (RU)](https://tlinks.run/tlbootcamp)
+
+# What is it about?
+
+The concepts of "Team Lead", "Tech Lead", "Tech Manager" and others are very stretchy. Companies solve different tasks, these roles have different responsibilities. Besides, various leadership styles, differences in personal skills, and much more are put in the picture. But isn't that what happens to the developers? Different companies, different tasks, again, different nature of tasks and responsibilities.
+
+We interviewed several dozen large and small companies, found out what they expect from Team Leads and how they interact with them. As a result, we were able to form a single model of basic teamwork competencies that covers all the cases studied.
+
+This model can be used in any way you like - to come up with your development plan, to create job descriptions in companies, to write a job description or to conduct interviews. Keep in mind it is likely you don't need all the branches of potential development - and this is fine.
+
+> 🚨 **Important:** Depending on a company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all the areas of the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
+
+The work is still in progress, so feel free to publish an Issue, drop your pull request and join the discussion [in Telegram chat, RU](https://tlinks.run/tlbootcamp).
+
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
+2. Go through all branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
```

Comment:
```suggestion
2. Go through all the branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
```
?

Reply:
Redundant, imho. That's definitely not a grammatical error and I don't see a reason to add more words here.